rotkehlchen/logging.py

```diff
@@ -214,3 +214,21 @@ def configure_logging(args: argparse.Namespace) -> None:
         logging.getLogger('urllib3').setLevel(logging.CRITICAL)
         logging.getLogger('urllib3.connectionpool').setLevel(logging.CRITICAL)
         logging.getLogger('substrateinterface.base').setLevel(logging.CRITICAL)
+        # TODO VN: remove
```

Comment:
I hope to remember XD. I've found this very useful and I have this block saved in my notes.